### PR TITLE
Feature: Move html5 location config into it's own module

### LINKED ISSUE #40: Move HTML5 location config into it's own module

### PR DESCRIPTION
**Pull request by edwardoparearyee**
Fixes #40

### CODE REVIEW

**Comment by jamesjwarren** [comment]
@edwardoparearyee is there a particular reason for doing this, or just separation?

**Comment by edwardoparearyee** [comment]
@jamesjwarren basically separation, ideally there should be no extra logic in our `app.js` file and we can clearly see what each bit of code is responsible for instead of placing random bits of unrelated code all in the `run` section of `app.js`.

If we wanted to remove html5 locations we can simply just remove the module from the `app.js` file instead of trying to figure out what code is responsible for html5 locations etc...

Basically trying to get rid of something like this: https://github.com/thisissoon/EightRoads-Frontend/blob/develop/app/js/app.js

**Comment by jamesjwarren** [comment]
@edwardoparearyee cool, sounds good :+1:
